tests/assets/ansible/roles/caasp/tasks/main.yaml

```diff
@@ -0,0 +1,8 @@
+---
+- name: Add repos
+  include_tasks: '{{ item }}'
```

Comment:
Wouldn't be `include_tasks: "{{ ansible_os_family }}.yaml"` enough?

Reply:
Well it would now, but it is ready for easy extension.

Reply:
hm. It makes things more complicated and the loop is not needed at all. but not blocking imo